openpype/hosts/max/plugins/load/load_model.py

```diff
@@ -87,6 +87,7 @@ def update(self, container, representation):
                     rt.Select(abc_con.Children)
                     for abc_obj in abc_con.Children:
                         abc_obj.source = path
+        self.log.debug(container)
```

Comment:
Pretty sure this max loader change should not be in this PR ;)
```suggestion
```

Reply:
It shouldn't be here